django/conf/__init__.py

```diff
@@ -91,21 +91,10 @@ def __init__(self, settings_module):
 
         mod = importlib.import_module(self.SETTINGS_MODULE)
 
-        tuple_settings = (
-            "ALLOWED_INCLUDE_ROOTS",
-            "INSTALLED_APPS",
-            "TEMPLATE_DIRS",
-            "LOCALE_PATHS",
-        )
         self._explicit_settings = set()
         for setting in dir(mod):
             if setting.isupper():
                 setting_value = getattr(mod, setting)
-
-                if (setting in tuple_settings and
-                        isinstance(setting_value, six.string_types)):
-                    raise ImproperlyConfigured("The %s setting must be a tuple. "
-                            "Please fix your settings." % setting)
```

Comment:
Actually, it might make sense to keep this check here. It's more of a "not a string" test than a "must be a tuple". We could just say "must be a list". (even though tuples are allowed too.)